functions/check_netlify_site.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+const axios = require("axios");
+const serverUtils = require("../functions-common/server_utils.js");
+const netlifySiteApi = "https://api.netlify.com/api/v1/sites";
+
+/**
+ * Support the endpoint /api/check_netlify_site
+ * Check if the GitHub repository defined via process.env.WORDLES_REPO_OWNER & process.env.WORDLES_REPO_NAME
```

Comment:
Would be helpful to explain the strategy here - it seems we make use of the https://api.netlify.com/api/v1/sites endpoint, and search through its entire response to see whether the expected repo is found in the build_settings field. Helpful also to link to the API documentation here.